WordPress/src/main/java/org/wordpress/android/ui/layoutpicker/LayoutsItemViewHolder.kt

```diff
@@ -1,24 +1,29 @@
 package org.wordpress.android.ui.layoutpicker
 
+import android.os.Bundle
+import android.os.Parcelable
 import android.view.LayoutInflater
 import android.view.ViewGroup
 import android.widget.TextView
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerView.OnScrollListener
 import org.wordpress.android.R
 
 /**
  * Modal Layout Picker layouts view holder
  */
 class LayoutsItemViewHolder(
     parent: ViewGroup,
-    private val prefetchItemCount: Int = 4
+    private val prefetchItemCount: Int = 4,
```

Comment:
I think we can remove the `,` at the end of the line

Reply:
removed, thanks